azure-pipelines.yml

```diff
@@ -519,7 +519,7 @@ stages:
             -DskipTests=true
           publishJUnitResults: false
           javaHomeOption: 'JDKVersion'
-          jdkVersionOption: '1.11'
+          jdkVersionOption: '1.17'
```

Comment:
We get

> Unhandled: Failed to find the specified JDK version. Please ensure the specified JDK version is installed on the agent and the environment variable 'JAVA_HOME_17_X64' exists and is set to the location of a corresponding JDK or use the [Java Tool Installer](https://go.microsoft.com/fwlink/?linkid=875287) task to install the desired JDK.

Shall we use the [JavaToolInstaller@0 task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/java-tool-installer-v0?view=azure-pipelines&viewFallbackFrom=azure-devops) or do we need to update the base image, @andrei-epure-sonarsource ?

Reply:
This was fixed by https://github.com/SonarSource/re-ci-images/pull/153